tools/cli/commands/docker.js

```diff
@@ -801,9 +817,39 @@ export function dockerDefine( yargs ) {
 					command: 'jt-config',
 					description: 'Set jurassic tube config',
 					handler: argv => execJtCmdHandler( argv ),
+				} )
+				.command( {
+					command: 'monorepo',
+					description: 'Run commands in monorepo container',
+					builder: yargCmd =>
+						defaultOpts( yargCmd ).option( 'cmd', {
+							alias: 'c',
+							describe: 'Command to run',
+							type: 'string',
+							demandOption: true,
+						} ),
+					handler: argv => {
+						const opts = buildComposeFiles().concat( [
+							'run',
+							'--rm',
+							'monorepo',
+							'bash',
+							'-c',
+							argv.cmd,
+						] );
+
+						const envOpts = buildEnv( argv );
+						composeExecutor( argv, opts, envOpts );
```

Comment:
This seems like it duplicates the intent of your `tools/docker/bin/monorepo` script, but does so with its own code that goes via `docker compose` instead of running the script.

At a quick search I'm not seeing anything that does `jetpack docker monorepo` instead of running `tools/docker/bin/monorepo`. Maybe we can drop this subcommand and `tools/docker/docker-compose-monorepo.yml` entirely? Or if we want to keep the subcommand, maybe we can still drop `tools/docker/docker-compose-monorepo.yml` and have it run `tools/docker/bin/monorepo` directly instead?

Reply:
Dropping the subcommand; it was left over from an aborted idea. 0bf7446db8